input/fsh/profiles/FinnishSchedulingPractitionerRole.fsh

```diff
@@ -0,0 +1,22 @@
+Profile: FinnishSchedulingPractitionerRole
```

Comment:
There's probably nothing scheduling specific here?
We should change to use the profile from base profiles.